meinberlin/apps/budgeting/api.py

```diff
@@ -8,11 +11,45 @@
 from .serializers import ProposalSerializer
 
 
+# To be changed to a more general IdeaPagination, when using
+# pagination via rest api for more idea lists
+class BudgetPagination(PageNumberPagination):
+    page_size = 15
+
+    def get_next_page(self):
+        if self.page.has_next():
+            return self.page.next_page_number()
+        else:
+            return None
+
+    def get_previous_page(self):
+        if self.page.has_previous():
+            return self.page.previous_page_number()
+        else:
+            return None
+
+    def get_paginated_response(self, data):
+        return Response({
+            'status': True,
+            'results': data,
+            'meta': {
+                'results_count': self.page.paginator.count,
+                'current_page': self.page.number,
```

Comment:
You should know this in react, because that is also in the url,  you get the data from?